cli/version/version.go

```diff
@@ -41,16 +129,48 @@ func BuildVersionString() string {
 		}
 	}
 
-	versionStr := fmt.Sprintf("v%s", version)
-	versionParts = append(versionParts, versionStr)
+	return formatVersion(cliVersionTmpl, map[string]string{
+		"Title":   cliVersionTitle,
+		"Version": version,
+		"OS":      runtime.GOOS,
+		"Arch":    runtime.GOARCH,
+		"Commit":  gitCommit,
+	})
+}
+
+func printCliVersion() {
+	fmt.Println(BuildCliVersionString())
+}
 
-	osArchStr := fmt.Sprintf("%s/%s", runtime.GOOS, runtime.GOARCH)
-	versionParts = append(versionParts, osArchStr)
+func PrintVersionString(projectPath string, projectPathIsSet bool, showRocksVersion bool) error {
+	printCliVersion()
+	if err := printCartridgeVersion(projectPath); err != nil {
```

Comment:
I suggest modifying this error message in the case when `showRocksVersion` is set. User should see something like "Failed to get Cartridge and other rocks versions: %s".